src/components/domain/ProfileForm/index.tsx

```diff
@@ -106,14 +114,20 @@ const ProfileForm = ({
       <form>
         <Center>
           <Spacer gap="xs" type="vertical">
-            <Upload style={{ textAlign: "center" }}>
-              <Avatar
-                isEdit
-                src={profileImage}
-                shape="circle"
-                __TYPE="Avatar"
-              />
-            </Upload>
+            <UploadableArea inputRef={profileImageRef}>
+              {(file: File, fileSrc: string) =>
+                fileSrc ? (
+                  <Avatar isEdit src={fileSrc} shape="circle" __TYPE="Avatar" />
+                ) : (
+                  <Avatar
+                    isEdit
+                    src={profileImage}
+                    shape="circle"
+                    __TYPE="Avatar"
+                  />
+                )
```

Comment:
`p4` 이렇게 리팩토링 하면 어떨까요?

```suggestion
               <Avatar isEdit src={isEdit ? fileSrc : profileImage} shape="circle" __TYPE="Avatar" />
```

Reply:
[3433ad9](https://github.com/slamapp/client/pull/55/commits/3433ad9322e5fd3b8ba76513179c77ac2c78de0f)에서 수정 완료했습니다!